### PR TITLE
fix(toast): Fix for toast showing black border when animating,

### PR DESCRIPTION
**Pull request by sambhav-gore**
#191

### CODE REVIEW

**Comment by dami-gg** [comment]
Please remove the animations, we agreed in the past that those have to be customised by the apps using dress code, we can't impose any value

**Comment by sambhav-gore** [comment]
Oh that is a mistake, I added to simulate the issue. Will remove it and re-commit.

File: src/styles/molecules/_toast.scss [hunk 1]
```diff
@@ -35,13 +35,20 @@
...
-    border-style: solid;
+    border-style: none;
     border-radius: $dc-radius50;
     font-size: $dc-font50;
     font-weight: $dc-bold-font-weight;
     line-height: 1.2;
     text-align: center;
     text-transform: uppercase;
+
+    &--success,
```
Comment (rbarilani) [inline src/styles/molecules/_toast.scss]: I think this breaks a little bit the concept of not relying of class names and give the possibility to the end user to use mixins to compose the style of a component. @sambhav-gore WDYT?

Comment (sambhav-gore) [inline src/styles/molecules/_toast.scss]: I agree, I will move the changes to the mixins.

**Pull request merged**